core/src/test/kotlin/com/exactpro/th2/read/db/app/DataBaseReaderIntegrationTest.kt

```diff
@@ -81,6 +82,7 @@ import java.time.temporal.ChronoUnit
 @IntegrationTest
 @TestInstance(TestInstance.Lifecycle.PER_CLASS)
 internal class DataBaseReaderIntegrationTest {
+    private val connection = mock<Connection> { }
```

Comment:
note: I believe we can use the real connection here as we have MySQL running in container